packages/cli/package.json

```diff
@@ -9,19 +9,12 @@
             "default": "./build/website.js"
         }
     },
+    "files": [
+        "build"
+    ],
     "dependencies": {
         "@svelte-add/core": "workspace:*"
     },
-    "peerDependencies": {
-        "@svelte-add/bootstrap": "workspace:^",
-        "@svelte-add/bulma": "workspace:^",
-        "@svelte-add/mdsvex": "workspace:^",
-        "@svelte-add/routify": "workspace:^",
-        "@svelte-add/tailwindcss": "workspace:^",
-        "@svelte-add/threlte": "workspace:^",
-        "@svelte-add/storybook": "workspace:^",
-        "svelte-add": "workspace:^"
-    },
```

Comment:
It's been a long time since I wrote this, but if I recall correctly this was added to make the changeset's work as expected. I initially added them as devDependencies so that they can properly be bundled, but [`changesets` refuse to work with those](https://github.com/changesets/changesets/issues/944)

At that time I couldn't find any proper workaround in the mentioned issue, and it does not look like someone has figured out something yet.

Reply:
makes sense!